lib/rules/landmark-main-is-top-level.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "landmark-main-is-top-level",
+  "selector": "main,[role=main]",
+  "tags": [
+	"best-practice"
+  ],
+  "metadata": {
+    "description": "Each main landmark in a document must not be nested in another landmark",
+    "help": "The main landmark in this document should not be contained within another landmark"
```

Comment:
This is too wordy and won't fit well into the extension UI. I'd suggestion: "The main landmark should not be in another landmark"